src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,108 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    public Book() {
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @Column(nullable = false)
+    private String Genre;
+    @Column(nullable = false)
+    private String Author;
+    @Column(nullable = false)
+    private String Image;
+    @Column(nullable = false)
+    private String Title;
+    @Column(nullable = false)
+    private String Subtitle;
+    @Column(nullable = false)
+    private String Publisher;
+    @Column(nullable = false)
+    private String Year;
+    @Column(nullable = false)
+    private int Pages;
+    @Column(nullable = false)
+    private String isbn;
```

Comment:
In these cases where the fields need to be unique ,I would not just add @column (nullable = false)
I would also add the following:
@column (nullable = false, unique = true)